analyzers/src/SonarAnalyzer.Common/Helpers/KnownType.cs

```diff
@@ -547,6 +547,10 @@ private KnownType(SpecialType specialType, string typeName)
 
         public string ShortName => shortName.Value;
 
+        public string ShortNameWithoutAttributeSuffix => shortName.Value == nameof(Attribute)
```

Comment:
you are adding an attribute-specific property in a class which is meant for any kind of type... I have an itch about it

Reply:
see my comment on AttributeSyntaxExtensions.cs   - probably it should be there